content/well-architected-framework/docs/docs/define-and-automate-processes/automate/cicd.mdx

```diff
@@ -32,4 +32,5 @@ In this section of Automate your workflows, you learned how to implement CI/CD p
 Visit the following documents to learn more about the automation workflow:
 
 - [Automate testing](/well-architected-framework/define-and-automate-processes/automate/testing) - Implement automated testing in your CI/CD pipeline
-- [Automate deployments](/well-architected-framework/define-and-automate-processes/automate/deployments) - Deploy applications through your CI/CD pipeline 
\ No newline at end of file
+- [Automate deployments](/well-architected-framework/define-and-automate-processes/automate/deployments) - Deploy applications through your CI/CD pipeline 
+- [Running Terraform in automation](/terraform/tutorials/automation/automate-terraform)
```

Comment:
can we add a short description here so it's consistent with the others?

out of scope for this PR, but we should prob rename that tutorial to "Run Terraform in automation" or even "Automate Terraform workflows"